### ISSUE TITLE
Error in job parse_tables

### ISSUE THREAD

**Issue by arumakanorat** [mpicbg-scicomp/tissue_miner#35]
Hi again,
When I have newly started TM again with 21 images, I got a below error message after loading the images. It looks loading error of dividing_cells.tif, however, there are apparently has dividing_cells.tif in each folder. When you have a chance could you please check this out? 
Sample name: Trial_images_LAmCherry_000 to _021
Thanks,
Taro

```
Parsing /movies/Trial_images_LAmCherry//Segmentation/Trial_images_LAmCherry_001/dividing_cells.tif...
DeformingTissue::parseDivisionsFile: Did not find any combination of mother-daughter pairs:
DeformingTissue::parseDivisionsFile: Mothers: 
DeformingTissue::parseDivisionsFile: Daughters: (271, 305) (15691036); (326, 277) (14862779); 
Could not load /movies/Trial_images_LAmCherry//Segmentation/Trial_images_LAmCherry_001/dividing_cells.tif!
Deleting states...
  0% [__________________________________________________________________________ 50% [##################################################________________________100% [##########################################################################                                                                                Error in job parse_tables while creating output file dbTablesFromParser/vertex_in_frame.dat.
RuleException:
CalledProcessError in line 51 of /tissue_miner/workflow/tm.snkmk:
Command 'imageParser $(dirname /movies/Trial_images_LAmCherry) $(basename /movies/Trial_images_LAmCherry) %03d' returned non-zero exit status 1
  File "/tissue_miner/workflow/tm.snkmk", line 51, in __rule_parse_tables
  File "/usr/lib/python3.4/concurrent/futures/thread.py", line 54, in run
Job failed, going on with independent jobs.
Exiting because a job execution failed. Look above for error message
```

**Comment by etournay** [comment]
Hi Taro,

This message indicates that there is an inconsistency in the tracking. Have you used TissueAnalyzer to track cells ? Which version ?

**Comment by arumakanorat** [comment]
Hi Raphael,

I used TissueAnalyzer for the tracking and the version is v1.0.

**Comment by etournay** [comment]
All right. That's the first time I see such a message. The numbers (271, 305) and (326, 277) are the coordinates of the daughter cells in frame 001. Could you crop this region for images 000 and 001 and share them on git hub? Oherwise you could directly email me @pasteur.fr. Best. Raphael

**Comment by arumakanorat** [comment]
Sure, here is the raw dividing_cells.tif files. I've put _000 and _001 to the end of original file name. Please find them via below DB link.
`https://www.dropbox.com/sh/4m5sakw1egrzlsi/AAC4vpsVOupC0oAaHu7GDS7Wa?dl=0`

Thanks
Taro

**Comment by etournay** [comment]
I tried to reproduce the problem with your data set but I works fine. 
Have you tried to correct the tracks in TA? Could you send the file tracked_cells_resized.tif for each of those images ?

**Comment by arumakanorat** [comment]
Thanks! That's great. I might done something wrong in TA. Yes, I did correct the track. Those tracked_cells_resized.tiff are in same DB folder.

**Comment by etournay** [comment]
That's it. 

Once tracks are manually corrected you need to update the tracking properly. As TA isn't based on a relational database, it is sometime hard to do things consistently. Best would be to ask Benoît Aigouy (IBDM, Marseille) to make a tutorial on this topic because it is sometime just impossible to repair lineages if you did a wrong manipulation. I'm not sure you can now rescue your tracking. I know it is a tedious thing to correct tracks. At first I would recommend not to correct tracks before you learn exactly how to do it ! Above a certain number of cells it is anyway impossible to do so manually. Best is to optimize the parameters of tracking (dynamics tissue) to get a nearly perfect tracking.

Good luck, 
Raphael

**Comment by arumakanorat** [comment]
That's exactly I felt on TA. Although I still could not figure out how I correct the tracked cells on TA, but I could rescue my data by following steps. Deleting all files except for handCorrection.tif in the individual image folder and then re-run the TA. I also got the same error on TM when I used the data which has correct the cell divisions on TA. But those things are probably overcome by optimizing the parameters that you told.
Thanks so much for taking your time to help! 
Taro
